src/bp/common/telemetry.ts

```diff
@@ -0,0 +1,43 @@
+import uuid from 'uuid'
+
+export type TelemetryEvent = Schema & {
+  event_type: string
+  event_data: TelemetryEventData
+}
+
+export type TelemetryEventData = Dic<any> & {
+  schema: string
+}
+
+export interface ServerStats {
+  externalUrl: string
+  botpressVersion: string
+  clusterEnabled: boolean
+  os: string
+  bpfsStorage: string
+  dbType: string
+  machineUUID: string
+  fingerprint: string | null
+  license: {
+    status: string
+    type: string
+  }
+}
+
+export interface Schema {
+  timestamp: Date
+  uuid: string
+  schema: string
+  source: string
+  server: ServerStats
+}
+
+export const getSchema = (server: ServerStats, source: string): Schema => {
```

Comment:
can you change this to either getFormattedSchema or formatSchema

Reply:
went with buildSchema in 05d500738a387dfc6ad38de1ed757fc925690cc5